navi.plugin.fish

```diff
@@ -1,21 +1,17 @@
-function navi-widget -d 'Call navi'
-  set -q NAVI_USE_FZF_ALL_INPUTS; or set -l NAVI_USE_FZF_ALL_INPUTS "true"
-  begin
-    navi --print | while read -l r; set result $result $r; end
-
-    if [ -n "$result" ]
-      echo $result
-
-      # Remove last token from commandline.
-      commandline -t ""
+function navi_key_bindings
```

Comment:
Why are you wrapping everything inside this fn?

I tried running `source $NAVI_HOME/navi.plugin.fish` and it didn't seem to work. Removing this line and the last one makes it work.

Reply:
I adapted the script from the fzf fish script. The bindings are enabled by a line in ~/.config/fish/functions/fish_user_key_bindings.fish

```
function fish_user_key_bindings
  fzf_key_bindings
  navi_key_bindings
end
```

Otherwise they are always enabled. Not a big deal though, you can also delete the file if you want it disabled.